src/v2/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -85,10 +86,14 @@ export class TransactionDetailsSummaryItem extends React.Component<
             <Flex flexDirection="column" mr="auto">
               <Text variant="sm" color="blue100">
                 Congratulations! This artwork will be added to your Collection
-                once the gallery ships the order.
+                once the gallery confirms the order.
               </Text>
               <Text variant="sm">
-                View and manage your Collection in the Artsy App.
+                View and manage all artworks in your Collection{" "}
+                {!isEigen ? "on the Artsy app." : "through your "}
+                {isEigen && (
+                  <RouterLink to={"artsy:///my-profile"}>profile</RouterLink>
```

Comment:
I am not sure if this is the way to navigate in Eigen. If there is any other way let me know I will change it.

Reply:
```suggestion
                  <RouterLink to={"artsy:///my-profile"}>profile.</RouterLink>
```

Reply:
We use deep linking for iOS and Android and the web routes usually match the [routes in the app](https://github.com/artsy/eigen/blob/0f5bc8a844cccea4fc1eccee81814af30df97068/src/lib/navigation/routes.tsx#L118).